hcl2template/types.build.hcp_packer_registry.go

```diff
@@ -0,0 +1,55 @@
+package hcl2template
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	packerregistry "github.com/hashicorp/packer/internal/packer_registry"
+)
+
+type HCPPackerRegistryBlock struct {
+	// Bucket description
+	Description string
+	// Bucket labels
+	Labels map[string]string
+
+	HCL2Ref
+}
+
+func (b *HCPPackerRegistryBlock) WriteBucketConfig(bucket *packerregistry.Bucket) {
```

Comment:
Naming NIT: 
```suggestion
func (b *HCPPackerRegistryBlock) configureFromBucketConfig(bucket *packerregistry.Bucket) {
```

Reply:
Hmm the newly suggested name sounds like is using the bucket to configure something but is actually the other way around.

Reply:
maybe a "To" is missing in the original one, like, `WriteToBucketConfig`

Reply:
right !

Reply:
I think confused the order here, as generally I like it more when the one on the right writes to the one on the left.

So I like more stuff like `x.SetupFrom(y)` than `y.WriteTo(x)`. Please do as you wish; this would be a personal preference :p

Reply:
I did SetupFrom but then there was a cyclic dependency. I didn't want to change too many things because I would change much of Wilken's logic that is still in progress. I don't want to make it hard. 
I'm going to keep WriteTo for now and once it is merged with Wilken's branch and the job is done we refactor the code.